e2e/test/scenarios/models/models-metadata.cy.spec.js

```diff
@@ -78,6 +80,14 @@ describe("scenarios > models metadata", () => {
       cy.findAllByTestId("header-cell")
         .should("contain", "Subtotal")
         .and("not.contain", "Pre-tax");
+
+      // Ensure back navigation works correctly metabase#55162
```

Comment:
Let's also add the issue tag in test name - it should decrease probability of accidental removal.